problems/stack/stack.py

```diff
@@ -5,13 +5,19 @@ class Stack():
     def __init__(self):
         self.stack = []
 
+    def isEmpty(self):
+		return True if len(self.stack) == 0 else False
```

Comment:
this could be shortened to

```python
def is_emtpy(self):
  return len(self.stack) == 0
```

Also note, that method names in Python are usually in `snake_case` instead of `camelCase`. 😉